bigquery/src/main/java/com/google/cloud/hadoop/io/bigquery/BigQueryHelper.java

```diff
@@ -135,25 +168,30 @@ public void importFromGcs(
       String projectId,
       TableReference tableRef,
       @Nullable TableSchema schema,
+      @Nullable TimePartitioning timePartitioning,
       @Nullable String kmsKeyName,
       BigQueryFileFormat sourceFormat,
       String writeDisposition,
       List<String> gcsPaths,
       boolean awaitCompletion)
       throws IOException, InterruptedException {
     logger.atInfo().log(
-        "Importing into table '%s' from %s paths; path[0] is '%s'; awaitCompletion: %s",
+        "Importing into table '%s' from %s paths; path[0] is '%s'; awaitCompletion: %s;  TimePartitioning: {}",
         BigQueryStrings.toString(tableRef),
         gcsPaths.size(),
         gcsPaths.isEmpty() ? "(empty)" : gcsPaths.get(0),
-        awaitCompletion);
+        awaitCompletion,
+        timePartitioning != null ? timePartitioning : "null");
 
     // Create load conf with minimal requirements.
     JobConfigurationLoad loadConfig = new JobConfigurationLoad();
     loadConfig.setSchema(schema);
     loadConfig.setSourceFormat(sourceFormat.getFormatIdentifier());
     loadConfig.setSourceUris(gcsPaths);
     loadConfig.setDestinationTable(tableRef);
+    if(timePartitioning != null) {
+      loadConfig.setTimePartitioning(timePartitioning);
```

Comment:
There no need for if condition - it should be safe to pass `null` value to `setTimePartitioning`